webcompat/helpers.py

```diff
@@ -583,8 +583,8 @@ def is_blacklisted_domain(domain):
     # see https://github.com/webcompat/webcompat.com/issues/1141
     # see https://github.com/webcompat/webcompat.com/issues/1237
     # see https://github.com/webcompat/webcompat.com/issues/1627
-    spamlist = ['qiangpiaoruanjian', 'cityweb.de', 'coco.fr']
-    return domain in spamlist
+    spamlist = ['www.qiangpiaoruanjian.cn', 'mailmanager.cityweb.de', 'www.coco.fr']
```

Comment:
`webcompat/helpers.py:586:80: E501 line too long (84 > 79 characters)`

Should I break it into 2 lines?